src/Common/ErrorPopup/ErrorPopup.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ReactElement, useEffect, useContext } from "react";
+import CloseIcon from "../../../public/assets/icons/CloseIcon";
+import styles from "./ErrorPopup.module.scss";
+import { useTranslation } from "next-i18next";
+import { QueryParamContext } from "src/Layout/QueryParamContext";
+import { SerializedError } from "@planet-sdk/common";
+import { ThemeContext } from "styles/themeContext";
+
+export default function ErrorPopup(): ReactElement {
+  const { t, ready } = useTranslation(["common"]);
+  const { errors, setErrors } = useContext(QueryParamContext);
+  const { theme } = useContext(ThemeContext);
+
+  useEffect(() => {
+    let timer: NodeJS.Timeout;
+    if (errors) {
+      timer = setTimeout(() => {
+        setErrors(null);
+      }, 5000);
+    }
+    return () => {
+      clearTimeout(timer);
+    };
+  }, [errors]);
+
+  const handleRemoveError = (message: string) => {
+    if (errors) {
+      const updatedErrors = (errors as SerializedError[]).filter(
+        (err: SerializedError) => err.message !== message
+      );
```

Comment:
1. `errors` does not need to be explicitly annotated as `SerializedError[]` - TS will take care of this implicitly due to the `if (errors)` line narrowing the type.
2. `err` does not need to be explicitly annotated as `SerializedError` for similar reasons - TS will take care of it implicitly.

Reply:
fixed here
https://github.com/Plant-for-the-Planet-org/planet-donations/pull/351/commits/6f1c19be47713c89bced6639d2a804710b486c84